src/components/pageLayout/pageLayout.tsx

```diff
@@ -95,7 +96,16 @@ const PageLayout = ({
               breadcrumbsType="digitalCollections"
               breadcrumbsData={breadcrumbs || []}
             />
-            <TemplateAppContainer contentPrimary={children as JSX.Element} />
+            <Box
```

Comment:
I'd like the `TemplateAppContainer` component to be more flexible and allow you to pass in these values. Can you add a TODO to update this later?

I don't want to block the work but I want to move closer to a better `TemplateAppContainer`.